test-fields.ps1

```
@@ -0,0 +1,58 @@
+            fields       = $fields
+        }
+    } | ConvertTo-Json -Depth 10
+
+    try {
+        $bodyBytes = [System.Text.Encoding]::UTF8.GetBytes($body)
+        $response = Invoke-WebRequest -Uri $url -Method POST -ContentType "application/json;charset=UTF-8" -Body $bodyBytes -UseBasicParsing
+        Write-Host "  OK ($($response.StatusCode))" -ForegroundColor Green
+        return $true
+    } catch {
+        $status = $_.Exception.Response.StatusCode.value__
+        Write-Host "  FAIL (HTTP $status)" -ForegroundColor Red
```

PSScriptAnalyzer (1.25.0)

[warning] File 'test-fields.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)